api/src/test/java/org/pmiops/workbench/firecloud/FireCloudServiceImplTest.java

```diff
@@ -65,8 +63,7 @@
   @Qualifier(FireCloudConfig.SERVICE_ACCOUNT_BILLING_V2_API)
   private BillingV2Api billingV2Api;
 
-  @MockBean private HttpTransport httpTransport;
-  @MockBean private IamCredentialsClient iamCredentialsClient;
+  @MockBean private FirecloudApiClientFactory firecloudApiClientFactory;
```

Comment:
This will have to be ignored in tests at least, I think. In some cases I believe it is necessary, or at least more consistent to use this format to specify mocks.